src/API/FeedIssues.php

```diff
@@ -56,7 +56,7 @@ public function __construct() {
 	public function get_feed_issues( WP_REST_Request $request ) {
 
 		try {
-			$feed_id = Pinterest\ProductSync::get_registered_feed_id();
+			$feed_id = Pinterest\FeedRegistration::get_registered_feed_id();
```

Comment:
While we're updating this line, it would be better to do a full import at the top of the file so that we don't have a partial namespace here.

```suggestion
			$feed_id = FeedRegistration::get_registered_feed_id();
```

Reply:
I have blindly followed the structure of the code in this file already - I can ofc fix it.